mmdet/models/detectors/detection_transformer.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
The file name `detection_transformer` and the class name `TransformerDetector` are different. It's better to make them the same.